pyartcd/pyartcd/redis.py

```diff
@@ -0,0 +1,38 @@
+import os
+from string import Template
+
+import aioredis
+
+# Redis instance template, to be renderes with env vars
+redis = Template('${protocol}://:${redis_password}@${redis_host}:${redis_port}')
+
+
+def redis_url(use_ssl=True):
+    if not os.environ.get('REDIS_SERVER_PASSWORD', None):
+        raise RuntimeError('Please define REDIS_SERVER_PASSWORD env var')
```

Comment:
Maybe this should be a less generic or even custom exception. This might be a case where we allow an exception to be targetedly swallowed. Maybe even a RedisException or RedisRuntimeError or so?